README.md

```
@@ -0,0 +1,157 @@
+### Planned
+- JOSS publication
+- Vignettes with reproducible examples
+- Instrument calibration transfer methods
+
+---
+
+## Citation
+
+If you use `horizons` in your research, please cite:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)